src/app/Fake.Core.ReleaseNotes/Changelog.fs

```diff
@@ -346,7 +386,10 @@ module Changelog =
                 | "" -> "Changelog"
                 | h -> h
 
-            (sprintf "# %s\n\n%s\n\n%s" header description entries)
+            let references =
+                x.References |> List.map (fun reference -> reference.ToString()) |> joinLines
+
+            (sprintf "# %s\n\n%s\n\n%s\n\n%s" header description entries references)
```

Comment:
Maybe update to string interpolation?

Reply:
It was a `sprintf` before, so I did not feel the need to change it. But sure, if an interpolated string is preferred here, I am happy to change that.

Reply:
Addressed by b94624d203718ea1e0dc8c0298396dc3cd72701c.